packages/api-types/src/events.ts

```diff
@@ -113,6 +134,7 @@ export interface ApiEvent<Vessel = EventVessel> {
   loitering?: LoiteringEvent
   encounter?: EncounterEvent<Vessel>
   port?: PortEvent
+  port_visit?: PortVisitEvent
```

Comment:
we using snake_case here but camelCase in the rest of the properties, is this a typo ?

Reply:
No, it is not a typo. This must match the event property returned by the API otherwise the data for port visit found in the event.